src/modules/main/sections/DataTable.vue

```diff
@@ -94,31 +93,31 @@
 						</NcActionButton>
 						<NcActionButton v-if="canManageElement(table) "
 							:close-after-click="true"
-							@click="$emit('create-view')">
+							data-cy="dataTableCreateViewBtn" @click="$emit('create-view')">
 							<template #icon>
 								<PlaylistPlus :size="20" decorative />
 							</template>
 							{{ t('tables', 'Create view') + (isViewSettingSet ? '*' : '') }}
 						</NcActionButton>
-						<NcActionButton v-if="canManageTable(table)" :close-after-click="true" @click="$emit('create-column')">
+						<NcActionButton v-if="canManageTable(table)" :close-after-click="true" data-cy="dataTableCreateColumnBtn" @click="$emit('create-column')">
 							<template #icon>
 								<TableColumnPlusAfter :size="20" decorative title="" />
 							</template>
 							{{ t('tables', 'Create column') }}
 						</NcActionButton>
 
-						<NcActionCaption :title="t('tables', 'Integration')" />
+						<NcActionCaption :name="t('tables', 'Integration')" />
 						<NcActionButton v-if="canCreateRowInElement(table)"
 							:close-after-click="true"
-							@click="$emit('import', table)">
+							data-cy="dataTableBtn" @click="$emit('import', table)">
```

Comment:
Could we make that n
```suggestion
							data-cy="dataTableImportBtn" @click="$emit('import', table)">
```